### PR TITLE
Add a check, if 's' is blank, remove the line instead of replacing it in scoreboard…

### PR DESCRIPTION
**Pull request by RestartFU**
I'm not sure if we should really add that check in case you actually want to replace a line by nothing. In any case, I still think that we should have an easy to understand way to remove a line from the scoreboard

### CODE REVIEW

**Comment by Thunder33345** [comment]
-1 this feels "magical" and functionally overloading set to do more then setting
i am in favor of having something more straightforward like a remove function that takes in the int as the line to remove

**Comment by Sandertv** [comment]
I'd just introduce a Remove method.

**Comment by RestartFU** [comment]
I just made a new commit to my PR. I added a Remove method.

File: server/player/scoreboard/scoreboard.go [hunk 1]
```diff
@@ -51,9 +51,19 @@ func (board *Scoreboard) Set(index int, s string) (err error) {
...
 		return fmt.Errorf("index out of range %v", index)
 	}
-
 	// Remove new lines from the string
 	board.lines[index] = strings.TrimSuffix(strings.TrimSuffix(s, "\n"), "\n")
+
+	return nil
+}
+
+// Remove removes a specific line from the scoreboard
```
Comment (Sandertv) [inline server/player/scoreboard/scoreboard.go:60]: If you add a `.` at the end here this PR is ready to be merged.

Comment (RestartFU) [reply server/player/scoreboard/scoreboard.go]: done

**Review by Sandertv — APPROVED** [review]

**Comment by Sandertv** [comment]
Looks good, thank you!

**Pull request merged**